chain/jsonrpc/src/api/validator.rs

```diff
@@ -49,3 +55,73 @@ impl RpcFrom<GetValidatorInfoError> for RpcValidatorError {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::api::RpcRequest;
+    use near_jsonrpc_primitives::types::validator::RpcValidatorRequest;
+    use near_primitives::hash::CryptoHash;
+    use near_primitives::types::{BlockId, EpochId, EpochReference};
+
+    #[test]
+    fn test_serialize_validators_params_as_vec() {
+        let block_hash = CryptoHash::new();
+        let block_hash_string = CryptoHash::new().to_string();
+        let params = serde_json::json!([block_hash_string]);
```

Comment:
Remove `block_hash_string` and do this instead

```suggestion
        let params = serde_json::json!([block_hash.to_string()]);
```